packages/components/organisms/f-checkout/src/mixins/validations.mixin.js

```diff
@@ -10,6 +10,9 @@ export default {
         */
         isFieldEmpty (type, field) {
             return this.$v[type][field].$dirty && !this.$v[type][field].required;
+        },
+        focusField (type, field) {
```

Comment:
This name doesn't quite match what it does, I would call it `touchField` (calls the touch method) or `dirtyField` (makes the field dirty).

Reply:
I think in f-registration there is a method called `onFieldBlur` which might be a good way to replace this method and also `isFieldValid` above

Reply:
ah the `formFieldBlur` function? I actually had it like that originally but it ended up being a bit messier than the current solution - I might take another look this morn though actually, now I'm understanding things a bit more 😄